futures-executor/benches/poll.rs

```diff
@@ -28,27 +30,27 @@ fn task_init(b: &mut Bencher) {
         num: u32,
         task: Option<Waker>,
     };
+    impl Unpin for MyFuture {}
 
     impl Future for MyFuture {
-        type Item = ();
-        type Error = ();
+        type Output = ();
 
-        fn poll(&mut self, cx: &mut task::Context) -> Poll<(), ()> {
+        fn poll(mut self: PinMut<Self>, cx: &mut task::Context) -> Poll<Self::Output> {
```

Comment:
I see that before it was also `task::Context`, but the others are imported directly